ROADMAP.md

```diff
@@ -1,118 +1,68 @@
-Docker Engine Roadmap
-=====================
+Moby Project Roadmap
+====================
 
 ### How should I use this document?
 
 This document provides description of items that the project decided to prioritize. This should
-serve as a reference point for Docker contributors to understand where the project is going, and
-help determine if a contribution could be conflicting with some longer terms plans.
+serve as a reference point for Moby contributors to understand where the project is going, and
+help determine if a contribution could be conflicting with some longer term plans.
 
 The fact that a feature isn't listed here doesn't mean that a patch for it will automatically be
 refused (except for those mentioned as "frozen features" below)! We are always happy to receive
```

Comment:
no "frozen features" below, at the moment?

Reply:
they didnt really seem relevant to me any more - `exec` is really owned via containerd, and the registry 2.0 changes have happened. We unfroze all the other areas previously (build etc). Ah I should remove the mention, yes...